src/Client/app/(admin)/manage-users/page.tsx

```
@@ -32,104 +35,161 @@ const columns = [
   const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false)
+  const [selectedUser, setSelectedUser] = useState('')
+  const [status, setStatus] = useState<boolean>(true)
   const hasSearchFilter = Boolean(filterValue)
+  const hasRoleFilter = Boolean(roleFilter)
 
   useEffect(() => {
     const ManageUser = async () => {
       try {
-        const response = await api.get('Account/All')
+        const response = await api.get<ApiResponse>(
+          `/Account?pageIndex=${page}&pageSize=${rowsPerPage}${roleFilter ? '&role=' + roleFilter : ''}${
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)